apps/main/src/loan/components/PageCrvUsdStaking/Statistics/RevenueChartTooltip.tsx

```diff
@@ -52,26 +52,21 @@ const CustomTooltip = ({ active, payload }: TooltipProps<ValueType, NameType>) =
       }}
       elevation={2}
     >
-      <Typography variant="bodyMBold">{formatDate(toDate(timestamp), 'long')}</Typography>
+      <Typography variant="bodyMBold">{formatDate(timestamp, 'long')}</Typography>
       <Stack
         direction="column"
-        sx={{
-          marginTop: Spacing.sm,
-          padding: Spacing.sm,
-          gap: 1,
-          backgroundColor: Layer[2].Fill,
-        }}
+        sx={{ marginTop: Spacing.sm, padding: Spacing.sm, gap: 1, backgroundColor: Layer[2].Fill }}
       >
-        <DataSet label={t`APR`} value={proj_apy.toFixed(2) + '%'} lineColor={Color.Primary[500]} />
+        <DataSet label={t`APR`} value={Number(apyProjected).toFixed(2) + '%'} lineColor={Color.Primary[500]} />
```

Comment:
the values should be in numbers already

btw, I've repeated .toFixed(2) + '%' myself, maybe it would be nice to create a helper function